<launch>
  <include file="$(find realsense_ros_camera)/launch/camera.launch">
    <arg name="depth_width" value="640"/>
    <arg name="depth_height" value="480"/>
    <arg name="enable_depth" value="true"/>
  </include>

  <node pkg="rqt_image_view" type="rqt_image_view" name="depth_viewer" args="/camera/depth/image_raw" required="true"/>
</launch>
